tobac/examples/eric_test.py

```diff
@@ -0,0 +1,9 @@
+from tobac.themes import tint
+from copy import deepcopy
+track_params = deepcopy(tint.objects.default_params)
+
+nc_file_path = ('/home/rjackson/tracer-jcss/Tgrid_*.nc')
```

Comment:
it would be great if others could use the data and repeat the tests ... is there an option to share the data?